nullaway/src/main/java/com/uber/nullaway/GenericsCheck.java

```diff
@@ -0,0 +1,145 @@
+package com.uber.nullaway;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.util.ASTHelpers;
+import com.sun.source.tree.ParameterizedTypeTree;
+import com.sun.source.tree.Tree;
+import com.sun.tools.javac.code.Attribute;
+import com.sun.tools.javac.code.Type;
+import com.sun.tools.javac.tree.JCTree;
+import java.util.HashSet;
+import java.util.List;
+
+public class GenericsCheck {
```

Comment:
```suggestion
public class GenericsChecks {
```
Reads a bit better to me